crates/bevy_render/src/view/mod.rs

```diff
@@ -61,11 +61,12 @@ impl Plugin for ViewPlugin {
 /// # use bevy_app::prelude::App;
 /// # use bevy_render::prelude::Msaa;
 /// App::new()
-///     .insert_resource(Msaa { samples: 4 })
+///     .insert_resource(Msaa::from(MultiSampleLevel::Sample4))
 ///     .run();
 /// ```
 #[derive(Resource, Clone, ExtractResource, Reflect)]
 #[reflect(Resource)]
+#[non_exhaustive]
```

Comment:
Wha the reason for this addition?

Reply:
Only 1 and 4 are supported for now by wgpu: https://github.com/gfx-rs/wgpu/issues/1832

Reply:
But shouldn't this then be applied to the `MultiSampleLevel` enum (which would gain the new levels) and not `Msaa` struct (which shouldn't receive any new fields due to that)?

Reply:
yup this should be on the new enum

Reply:
Fixed